fineract-provider/src/main/java/org/apache/fineract/portfolio/loanaccount/domain/transactionprocessor/AbstractLoanRepaymentScheduleTransactionProcessor.java

```diff
@@ -209,7 +209,13 @@ public int compare(LoanRepaymentScheduleInstallment ord1, LoanRepaymentScheduleI
             }
         }
         reprocessInstallments(installments, currency);
-
+        if (charges != null) {
+            charges.forEach(charge -> {
+                if (charge.isInstalmentFee()) {
```

Comment:
this could happen for non-installment fee either...

Reply:
After a discussion we move to another way...